packages/app-mobile/components/NoteEditor/NoteEditor.tsx

```diff
@@ -15,14 +15,27 @@ export interface UndoRedoDepthChangeEvent {
 	redoDepth: number;
 }
 
+export interface Selection {
+	start: number;
+	end: number;
+}
+
+export interface SelectionChangeEvent {
+	selection: Selection;
+}
+
```

Comment:
It doesn't look like you need to export these two interfaces?

Reply:
These are exported because while `Note#body_selectionChange` currently has parameter `event: any`, when we transition to the Beta editor, we'll want to change its signature to `body_selectionChange(event: SelectionChangeEvent)`, like the `onBodyChange(event: ChangeEvent)` we currently have.

Additionally, `export`ing these two interfaces should make it clear that clients of `NoteEditor` can expect to interact with it through `SelectionChangeEvent`s.

Reply:
If it's better to not export them, I can make the change!